site/nginx.conf

```diff
@@ -21,10 +21,19 @@ http {
   ssl_protocols TLSv1 TLSv1.1 TLSv1.2; # Dropping SSLv3, ref: POODLE
   ssl_prefer_server_ciphers on;
 
-  log_format combined_real_ip '$http_x_real_ip - $remote_addr - $remote_user [$time_local] '
-                              '$scheme "$request" $status $body_bytes_sent '
-                              '"$http_referer" "$http_user_agent"';
-  access_log /var/log/nginx/access.log combined_real_ip;
+  log_format json-log escape=json '{'
+   '"message":"$scheme $request done in ${request_time}s: $status",'
+   '"response_status":$status,'
+   '"request_duration":$request_time,'
+   '"remote_address":"$remote_addr",'
+   '"x_real_ip":"$http_x_real_ip",'
+   '"request_start_time":"[$time_local]",'
```

Comment:
I don't think we want to bracket the time in the JSON format.

Reply:
Any reason it is [here](https://github.com/hail-is/hail/blob/1215c5152e60825c5f7e1ab700cb6ba31788ecc2/hail/python/hailtop/hail_logging.py#L33)

Reply:
I think I copied that from the internet.

Reply:
Ok I'll update the jsonlogger to match